### PR TITLE
Help screen and prefixing checkboxes

### PR DESCRIPTION
**Pull request by whot**
Three things I was missing when using uji recently:

- ability to press ? to get the list of key bindings, #10
- ability to quickly add SKIP to a test that I'm not running, #13 
- auto-advance to the next checkbox when I'm toggling it

cc @bentiss

### CODE REVIEW

File: uji.py [hunk 1]
```diff
@@ -846,6 +869,26 @@ def __init__(self, directory):
         self.cursor_offset = 0
         self.error = None
         self.show_filename_enabled = True
+        self.display_help = False
+
+        keymap = (
+            Keymapping('<ESC>', 'quit/exit help', flags=[KeymappingFlags.ACTIVE_IN_HELP], func=self.exit),
+            Keymapping('q', 'quit', flags=[KeymappingFlags.ACTIVE_IN_HELP], func=self.quit),
```
Comment (bentiss) [inline uji.py]: I tend to mentally associate `q` and `<ESC>`, meaning that in the help window I type `q` and leave entirely uji :(

File: uji.py [hunk 2]
```diff
@@ -1092,6 +1092,8 @@ def toggle(self):
         else:
             self.unmark()
 
+        self.next()
```
Comment (bentiss) [inline uji.py:1117]: Shouldn't this be called also after running a program?

It just feels wrong to have a different behavior, but OTOH, "running" doesn't mean that we are done with the test, just that we executed it.

And now that I come to it, should this be only valid for skip/pass/fail?

**Comment by bentiss** [comment]
No strong push back from my side. Code looks OK, just a couple of nitpicks.

File: uji.py (hunk 1 above)
Comment (whot) [reply uji.py]: fixed!

File: uji.py (hunk 2 above)
Comment (whot) [reply uji.py:1117]: yep, let's do that. Was missing because this patchset had to be rebased on top of the run patchset, so it was missing from there

**Pull request merged**